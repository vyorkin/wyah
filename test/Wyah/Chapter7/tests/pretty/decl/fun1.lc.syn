Decl "compose" (ELam (Var "g") (ELam (Var "f") (ELam (Var "x") (EApp (EVar (Var "f")) (EApp (EVar (Var "g")) (EVar (Var "x")))))))
