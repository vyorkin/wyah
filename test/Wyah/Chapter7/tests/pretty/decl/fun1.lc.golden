let compose = \g -> \f -> \x -> f (g x);